### ISSUE TITLE
Image url() from collectionFS return (ws://) prefix

### ISSUE THREAD

**Issue by radiegtya** [inProgress-team/react-native-meteor#51]
Image url() from collectionFS return (ws://) prefix

```
ws://localhost:3000/cfs/files/images/3xnScz4irQggxopBK?store=images
```

that's make error at my RN, although it can be simply resolve by replacing the String.

**Comment by radiegtya** [comment]
fixed by:

replacing ws:// with http:// or https://

example
```javascript
var image = Meteor.FSCollection('images').findOne(p.photos[0]);
image = image.url().replace('ws://', 'http://');
```

but I think this should be automatically replace from the react-native-meteor itself

**Comment by Mokto** [comment]
Yes you are right !

Fixing it right now. 


Thanks !

**Comment by Mokto** [comment]
Fixed in 1.0.0-beta27

**Comment by radiegtya** [comment]
ok thanks @Mokto
